### PR TITLE
[PortfolioProfitability] Add tests

### PR DESCRIPTION
**Pull request by Herklos**
Depends on #232

### CODE REVIEW

File: octobot_trading/data/portfolio_profitability.py [hunk 1]
```diff
@@ -130,16 +142,53 @@ async def get_average_market_profitability(self):
...
+        :return: the holdings ratio
+        """
+        currency_holdings: float = self.portfolio_manager.portfolio.get_currency_from_given_portfolio(currency,
+                                                                                                      PORTFOLIO_TOTAL)
+        currency_value: float = await self._evaluate_value(currency, currency_holdings)
+        return currency_value / self.portfolio_current_value if self.portfolio_current_value else 0
+
+    async def update_portfolio_current_value(self, portfolio, currencies_values=None, fill_currencies_values=False):
+        """
+        Update the current portfolio value
```
Comment (GuillaumeDSM) [inline octobot_trading/data/portfolio_profitability.py:175]: I think it should be more like: `Update the portfolio with current prices`

File: tests/data/test_portfolio_profitability.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    assert portfolio_profitability.profitability_diff == 0
+    assert portfolio_profitability.market_profitability_percent == 0
+    assert portfolio_profitability.initial_portfolio_current_profitability == 0
+    assert portfolio_profitability.portfolio_origin_value == 0
+    assert portfolio_profitability.portfolio_current_value == 0
+
+
+async def test_simple_handle_mark_price_update(backtesting_trader):
+    config, exchange_manager, trader = backtesting_trader
+    portfolio_profitability = exchange_manager.exchange_personal_data.portfolio_manager.portfolio_profitability
```
Comment (GuillaumeDSM) [inline tests/data/test_portfolio_profitability.py:41]: WIP ?

File: tests/data/test_portfolio_profitability.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        'BTC': 10,
+        'ETH': 5000,
+        'XRP': 0.1,
+        'USDT': 0.2
+    }
+
+
+def update_portfolio_balance(new_balance, exchange_manager):
+    exchange_manager.exchange_personal_data.portfolio_manager.portfolio.portfolio = new_balance
+    return new_balance
```
Comment (GuillaumeDSM) [inline tests/data/test_portfolio_profitability.py:234]: WIP ?

Comment (Herklos) [reply tests/data/test_portfolio_profitability.py:234]: Nope that's just a setter

**Pull request merged**